client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -189,12 +193,38 @@ export const SelectiveExtensionsBundle = ( {
 		};
 	} );
 
+	const { installedPlugins = [] } = useSelect( ( select ) => {
+		const { getInstalledPlugins } = select( PLUGINS_STORE_NAME );
+
+		return {
+			installedPlugins: getInstalledPlugins(),
+		};
+	} );
+
 	const installableExtensions = useMemo( () => {
 		return freeExtensions.filter( ( list ) => {
 			return ALLOWED_PLUGIN_LISTS.includes( list.key );
 		} );
 	}, [ freeExtensions ] );
 
+	const removeWCPayFromInstallableExtensions = () => {
+		installableExtensions.forEach( ( extensions ) => {
+			const { plugins } = extensions;
+			extensions.plugins = plugins.filter(
+				( plugin ) => plugin.key !== 'woocommerce-payments'
+			);
+		} );
+	};
+
+	if (
+		window.wcAdminFeatures &&
+		window.wcAdminFeatures.subscriptions &&
+		!! installableExtensions.length &&
+		installedPlugins.includes( 'woocommerce-payments' )
+	) {
+		removeWCPayFromInstallableExtensions();
```

Comment:
This won't be necessary if we filter the list by `is_installed` anyway.
For woocommerce-payments we might want to test if it's activated specifically though, especially if `subscriptions` is selected under `product_types`.

Reply:
I added this change in the commit 25f82f7